<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoAns</class>
 <widget class="QWidget" name="AutoAns">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>350</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>410</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AutoAns</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="LabPinImg">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>230</y>
     <width>130</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabLine">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>330</width>
     <height>1</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LabTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>6</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnLogout">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>150</width>
     <height>38</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>登出</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnLogin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>150</width>
     <height>38</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>50</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LnEdName">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>64</y>
     <width>190</width>
     <height>38</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>38</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="LnEdPasswd">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>118</y>
     <width>190</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="LnEdPin">
   <property name="geometry">
    <rect>
     <x>155</x>
     <y>170</y>
     <width>190</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>67</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabPasswd">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="LabPin">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>174</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>验证码：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="PrintBrow">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="LabSplitLn">
   <property name="geometry">
    <rect>
     <x>393</x>
     <y>12</y>
     <width>1</width>
     <height>326</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
